cli/cli.go

```diff
@@ -140,11 +140,16 @@ func GetCommands(logger utils.Log) []*clitool.Command {
 				if err != nil {
 					return
 				}
-				err = npmModule.CalcDependencies()
+				formatValue, filteredArgs, err := extractStringFlag(context.Args().Slice(), formatFlag)
 				if err != nil {
 					return
 				}
-				return printBuild(bld, context.String(formatFlag))
+				npmModule.SetNpmArgs(filteredArgs)
+				err = npmModule.Build()
+				if err != nil {
+					return
+				}
```

Comment:
The above line can be reduced to - 
```
				if err = npmModule.Build(); err != nil {
					return
				}
```
Let's implement this change for other commands in this file as well.